core/v2/types.go

```diff
@@ -161,7 +161,7 @@ func (b *BlobHeader) GetEncodingParams(blobParams *core.BlobVersionParameters) (
 	}, nil
 }
 
-type RelayKey uint16
+type RelayKey = uint16
```

Comment:
What's the consideration of changing to alias? The current one provides type safety which is often desired

Reply:
It's mostly to interoperate with chainio library which still lives in core and can't import v2 due to import cycles. 
So chainio library uses `uint16` instead of `RelayKey` type, and I updated this to type alias to convert the output from chainio without an explicit conversion method